server/openapi.yml

```diff
@@ -48,6 +48,33 @@ paths:
           $ref: "#/components/responses/Unauthorized"
         500:
           $ref: "#/components/responses/InternalServerError"
+  /users/{userId}:
+    get:
+      tags:
+        - user
+      summary: ユーザ情報取得API
+      description: ユーザIDでユーザ情報を取得する
+      parameters:
+        - name: userId
+          in: path
+          description: ユーザID
+          required: true
```

Comment:
これはpath parameterやったら必須なんやな 👀 

> Determines whether this parameter is mandatory. If the [parameter location](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#parameterIn) is "path", this property is REQUIRED and its value MUST be true. Otherwise, the property MAY be included and its default value is false.

https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#parameterObject